packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -16,6 +16,7 @@ export interface IEmailOptions {
   from?: string;
   text?: string;
   attachments?: IAttachmentOptions[];
+  inlineAttachments?: IAttachmentOptions[];
```

Comment:
@smbecker we already have a `disposition` flag passed in the `IAttachmentOptions` 
https://github.com/smbecker/novu/blob/fe8ebc1a57f18626b78340883871ba11a7252c52/packages/stateless/src/lib/template/template.interface.ts#L57

I would prefer if we have a single attachments array, with attachments of different types rather a new array.

What are your thoughts of those 2 approaches?

Reply:
Honestly, when I initially kicked off this branch, `cid` and `disposition` weren't available several months ago. I finally got some time to finish but I should have checked on recent changes. I'll rework the PR to take advantage of those.

Reply:
Sorry for the delay, I finally got back to this. Using the existing `cid` and `disposition` fields ended up being much cleaner.